### DISCUSSION TITLE
Duplicate Elastalert Rules

### DISCUSSION THREAD

**Discussion by BuffaloWill** [Security-Onion-Solutions/securityonion#8932] [Unsupported Versions] [answered]
Hi!

After making updates to my elastalert yml files in `/opt/so/rules/elastalert/` I find that I have duplicate rules running. For example, I will receive an email alert for the same rule twice; one with the old text and another email with the updated text. How can I remove these duplicated rules or triage this? The odd thing is I don't see the duplicate yml files inside the elastalert docker container.

**Comment by dougburks** [comment, marked as answer]
Have you restarted ElastAlert?

Have you checked the ElastAlert log for additional clues?
https://docs.securityonion.net/en/2.3/elastalert.html#diagnostic-logging

**Reply by BuffaloWill** [reply]
> Have you checked the ElastAlert log for additional clues?
Yes. That was my first check, the rule wasn't duplicated in the logs. 

> Have you restarted ElastAlert?
`so-elastalert-restart` look like it fixed it. Thank you!
